### ISSUE TITLE
IOS sync with Dropbox not succeeding

### ISSUE THREAD

**Issue by Steuma** [laurent22/joplin#404]
### Operating system

- iOS

### Application

- Mobile

Debug not available for IOS <-> Dropbox syncronize
First I have to say, that you provide a great productivity tool here. Thanks a ton.
Secondly I run into problems on synchronizing my iPhone6 with the dropbox data.
Not sure if it is a bug or just lack of information but since the 43 digit character code from
Dropbox cannot be transfered with Copy/Paste (that would be a big help ;-) ) I tried around
20 times now to key it into Joplin on my iPhone but I do get the same error message every time:

![grafik](https://user-images.githubusercontent.com/38250883/38558742-55341c58-3cd1-11e8-9d5b-5155b7a7593f.png)

I am gonna try to sync with WebDAV - I hope that will work out;-)

Cheers, Steuma

**Comment by laurent22** [comment]
Hello, this error message means that the code has already been used or is invalid (maybe was not copied correctly). So please check the code and try again.

I did not know it cannot be copied and pasted and that's indeed a major annoyance so I'll look into it.

**Comment by Steuma** [comment]
Thanks for the response, Laurent.
As for the copy/paste I am confused at the moment, since on: https://www.iphone-ticker.de/joplin-freie-notiz-loesung-fuer-ios-und-macos-124657/#comment-1000226 ( a german webpage unfortunatley), where I posted as well the user Peter claims in te comment section that he could do copy/paste. If indeed it is possible, I would be interested to learn why/where I did fail. 
Cheers, Steuma
